test/test_torch.py

```diff
@@ -8454,29 +8454,40 @@ def test_scatter_add_bool(self):
 
     def test_masked_scatter(self):
         with warnings.catch_warnings(record=True) as w:
```

Comment:
unconditionally catching the warnings and not checking them doesn't seem great -- we don't actually check that we raise warnings for uint8 and if we start raising warnings for bool no one will notice.